backend/src/test/java/org/eclipse/tractusx/semantics/registry/AssetAdministrationShellApiSecurityTest.java

```diff
@@ -665,7 +665,8 @@ public void testGetShellWithFilteredSpecificAssetIdsByTenantId() throws Exceptio
             AssetAdministrationShellDescriptor shellPayload = TestUtil.createCompleteAasDescriptor();
             shellPayload.setSpecificAssetIds(null);
             SpecificAssetId asset1 = TestUtil.createSpecificAssetId("CustomerPartId","tenantTwoAssetIdValue",List.of(jwtTokenFactory.tenantTwo().getTenantId()));
-            SpecificAssetId asset2 = TestUtil.createSpecificAssetId("CustomerPartId","tenantThreeAssetIdValue",List.of(jwtTokenFactory.tenantThree().getTenantId()));
+            //TODO: Can this happen in real life???
+            SpecificAssetId asset2 = TestUtil.createSpecificAssetId("CustomerPartId2","tenantThreeAssetIdValue",List.of(jwtTokenFactory.tenantThree().getTenantId()));
```

Comment:
yes, because the specificAssetId is not unique with key value. 
In older AAS version, we defined multiple time the same key value with different externalSubjectId